aspnetcore/blazor/fundamentals/routing.md

```diff
@@ -1902,6 +1920,116 @@ In the following `App` component example:
 > [!NOTE]
 > Not throwing if the cancellation token in <xref:Microsoft.AspNetCore.Components.Routing.NavigationContext> is canceled can result in unintended behavior, such as rendering a component from a previous navigation.
 
+## Handle location changes in a component
+
+`RegisterLocationChangingHandler` registers a handler to process incoming navigation events. The handler's context provided by `LocationChangingContext` includes the following properties:
```

Comment:
Do we need to use an `<xref:>` for `RegisterLocationChangingHandler` and `LocationChangingContext`?

Reply:
We do, but they won't be available for cross-linking until GA lands. I circle around and add them later.